src/main/java/com/nvs/store/repository/ProductRepository.java

```diff
@@ -8,6 +8,8 @@
 public interface ProductRepository extends JpaRepository<Product, Long> {
     Product getProductById(Long id);
 
-    Product  findTopByOrderByIdDesc();
+    Product findTopByOrderByIdDesc();
+
+    List<Product> findAll();
```

Comment:
I guess you dont need to specify this method explicitely - its part of jpa repository